crates/re_space_view_tensor/src/space_view_class.rs

```diff
@@ -152,21 +137,41 @@ impl SpaceViewClass for TensorSpaceView {
             view_property_ui::<TensorViewFit>(ctx, ui, view_id, self, state);
         });
 
+        // TODO(andreas): Listitemify
         if let Some((_, tensor)) = &state.tensor {
+            let slice_property = ViewProperty::from_archetype::<TensorSliceSelection>(
+                ctx.blueprint_db(),
+                ctx.blueprint_query,
+                view_id,
+            );
+            let slice_selection =
+                load_tensor_slice_selection_and_make_valid(&slice_property, tensor.shape())?;
+
             ui.separator();
             ui.strong("Dimension Mapping");
-            dimension_mapping_ui(ui, &mut state.slice.dim_mapping, tensor.shape());
-            let default_mapping = DimensionMapping::create(tensor.shape());
+            dimension_mapping_ui(ctx, ui, tensor.shape(), &slice_selection, &slice_property);
+
+            // TODO(andreas): this is a bit too inconsistent with the other UIs.
```

Comment:
In what way?

Reply:
I mean just scroll a few lines up: all the other UIs solve the general blueprint handling with generic buttons and menus, this one hacks in two extra buttons
I'll leave an additional note